student.rb

```diff
@@ -0,0 +1,12 @@
+class Student < Person
+  attr_reader :classroom
+
+  def initialize(name: 'Unknown', age: 0, parent_permission: true, classroom: 'Unknown')
+    super(name: name, age: age, parent_permission: parent_permission)
+    @classroom = classroom
+  end
+
+  def play_hooky
+    '¯\\(ツ)/¯'
+  end
+end
```

Comment:
- [ ] kindly add a method that serves as `belongs to side` of student to classroom like the code below
```
def classroom=(classroom)
      @classroom = classroom
      classroom.students.push(self) unless classroom.students.include?(self)
  end
```